src/main/java/com/epam/lab/paymentsystem/controller/CardController.java

```diff
@@ -91,6 +91,13 @@ public String getAccountPageUserCards(
     return ACCOUNT_PAGE;
   }
 
+  @GetMapping(value = "/myCards")
+  public String getMyCards() {
```

Comment:
Please, rename

Reply:
Also, I think it should be replaced to `IndexController` or new `TabBarController` since this functional is only proxy